core/controllers/admin.py

```diff
@@ -288,7 +288,13 @@ def post(self):
                 result = {
                     'version': version
                 }
-            elif action == 'update_feature_flag_rules':
+            else:
+                # The handler schema defines the possible values of 'action'.
+                # If 'action' has a value other than those defined in the
+                # schema, a Bad Request error will be thrown. Hence, 'action'
+                # must be 'update_feature_flag_rules' if this branch is
+                # executed.
+                assert action == 'update_feature_flag_rules'
```

Comment:
I'll add the necessary steps in the branch coverage wiki.

Reply:
Can you please do that before this PR is merged? I think it can be simple mention of the branch coverage and some explanation how to read the coverage reports and how to fix the issues.

Reply:
I think this is a pattern of the controller code that makes sense regardless of whether the code is tested. Therefore, I don't think the branch coverage wiki is the right place for this. Instead, this should go in a wiki page on how to write controllers (which I don't think exists)